packages/@monocdk-experiment/assert/package.json

```diff
@@ -47,7 +47,7 @@
     "@aws-cdk/cloudformation-diff": "0.0.0"
   },
   "peerDependencies": {
-    "constructs": "^3.0.4",
```

Comment:
We normally recommend to declare a pinned devDependency in order to ensure that the code gets tested with the minimal version required by the peer. Is this not the case here? Why?

Copying @nija-at and @NetaNir whom
I talked to about this.